layouts/partials/fragments/nav.html

```diff
@@ -81,3 +101,30 @@
     Top
   {{- end -}}
 </div>
+{{- if .Params.search }}
+<!-- this template is sucked in by search.js and appended to the search-results div above. So editing here will adjust style -->
+<script id="search-result-template-{{ .Name }}" type="text/x-js-template">
+  <div id="summary-${key}">
+    <h5><a href="${link}">${title}</a></h5>
```

Comment:
I would link the whole container and not just the title.
In more detail: Make the whole container for each search result clickable, but show only the title as actual link.